tests/float/tests/01-run.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    child.expect(u"Testing floating point arithmetics \(\d+ iterations\)...")
```

Comment:
This fails now (is this line [in the c-code also] necessary at all).

Reply:
it prevents from accidentally parsing the output of a previous run (output by board before calling `make reset`), so I think it makes sense to keep it.